fabric-datafixer-v1/src/main/java/net/fabricmc/fabric/datafixer/mixin/MixinPersistentState.java

```diff
@@ -0,0 +1,10 @@
+package net.fabricmc.fabric.datafixer.mixin;
+
+import org.spongepowered.asm.mixin.Mixin;
+
+import net.minecraft.world.PersistentState;
+
+@Mixin(PersistentState.class)
+public class MixinPersistentState {
+    // Add our mod's data versions to method_17919
```

Comment:
Imo the mod data version should go into each persistent state instance, as each persistence state shouldreally be touched by only one mod.

Reply:
I looked around and saw these are the classes that extend the PersistentState (Excluding the Manager):

ChunkUpdateState
ForcedChunkState
RaidManager
ScoreboardState
MapState
IdCountsState

The whole reason I initially added it into method_17919 (closer to writeStateToFile in functionality) is to keep all the DataVersions together in the same spot. This is also for the implementation because it will read the mod provided DataVersions in CompoundTag to tell the DataFixer what Schema version the data it is looking at is.